### ISSUE TITLE
Array plot with free field choice

### ISSUE THREAD

**Issue by kaloeffler** [N2-Survey/SurveyFramework#67]
@lheckmann I observed that besides the Likert plots there is another array question type without fixed choices. 
One example where this type occurs is for working times - e.g. how much time (percentage) do you spend on different tasks. The participants can add arbitrary integers to the categories which need to sum up to one.

Perhaps you could add an example plot and add some todos for that plot type?

**Comment by lheckmann** [comment]
> @lheckmann I observed that besides the Likert plots there is another array question type without fixed choices. One example where this type occurs is for working times - e.g. how much time (percentage) do you spend on different tasks. The participants can add arbitrary integers to the categories which need to sum up to one.
> 
> Perhaps you could add an example plot and add some todos for that plot type?

That is true, thank you for spotting it. Do you want to treat it in the same issue or make a new one for it?
And it means that we have to check (I guess in the central plot function) for the array plots if the answers are integers and if they are not, choose the Likert ones, if they are, choose the new plot function for integers.

**Comment by kaloeffler** [comment]
@lheckmann  I would suggest treating it in a different issue, since we either need at least some pre-processing to group the answers - e.g. create some "choices" - then I presume we could perhaps process it with the Likert plot functionality as well. Or we even might want to visualize this differently.

Concerning checking: I can only speak for the HeJu data set - here it's easy to check as no choices are available for these questions (`survey.get_choices(question_label)` returns None).
